src/reporter.cc

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 #include "benchmark/reporter.h"
-#include "minimal_leastsq.h"
```

Comment:
Yes, makes perfect sense. I have also added the include to console_reporter.cc, because function "GetBigOString" comes also from complexity.h.
On a different topic, I am not sure why after changing the doubles to 0.0, coveralls started to complain about coverage decreased.